File: OpenMP/hello3versomp.c
#include <stdio.h> 
#include  "omp.h"
int main(void) 
{ 
   char greetingsen[] = "Hello world!"; 
   char greetingsde[] = "Hallo Welt!";
   char greetingsfr[] = "Bonjour le monde!";
   int a;
   #pragma parallel 

   #pragma omp sections
   {
   #pragma omp section
   for ( a = 0; a < 100; a = a + 1 )
   	{
	printf("%s\n", greetingsen); 
   	}

   #pragma omp section
   for ( a = 0; a < 100; a = a + 1 )
   	{
	printf("%s\n", greetingsde); 
   	}

   #pragma omp section
   for ( a = 0; a < 100; a = a + 1 )
   	{
	printf("%s\n", greetingsfr); 
   	}
    }
return 0;
}

File: OpenMP/hello3versomp.f90
program hello
	implicit none 
        include "omp_lib.h"

	character(len=16) :: greetingsen
	character(len=16) :: greetingsde
	character(len=16) :: greetingsfr 
	integer :: a
	greetingsen = "Hello World!" 
	greetingsde = "Hallo Welt!"
	greetingsfr = "Bonjour le monde!"	

	!$omp parallel 
	!$omp sections

	!$omp section
	do a = 1, 100
	print *, greetingsen   
	end do

	!$omp section
	do a = 1, 100
	print *, greetingsde
	end do

	!$omp section
	do a = 1, 100
	print *, greetingsfr  
	end do

	!$omp end sections 
	!$omp end parallel
	
end program hello

